lib/src/signed_video_internal.h

```diff
@@ -144,6 +144,7 @@ struct _signed_video_t {
   bool is_golden_sei;  // Flag that tells if a SEI is a golden SEI
   bool using_golden_sei;  // Flag that tells if golden SEI prinsiple is used
   bool signing_started;
+  bool generate_first_sei;
```

Comment:
This name is not good. It indicates that you want to generate a first sei, but it becomes true after the first I-frame, hence all SEIs are look like being "first".
Try to find a name that better suites its purpose, for example, `is_first_triggered_signing`